SecurityCardManagement/README.md

```diff
@@ -79,12 +80,21 @@ returns:
 
 arguments: 
 
-	answerId: uint32 - function id of result callback	
+	answerId: uint32 - function id of result callback
 
 returns: 
 
 	state : string - result of operation
-	
+
+`createKeyForHmac` - create secret key for HMAC SHA256. This key is saved into Android keystore or iOS keychain and then is used by the app to sign APDU commands data fields. 
+
+arguments: 
+
+	answerId: uint32  - function id of result callback
+	p1:       bytes   - authentication password (128 bytes) 
+	cs:       bytes   - common secret(32 bytes)
+	sn:       uint192 - card serial number
+
```

Comment:
Mark that the function returns nothing like:
```
returns:
    void
```

Reply:
or maybe it can return status like `success` or fail reason.

Reply:
done